docs/install/workspaces.md

```diff
@@ -6,15 +6,34 @@ To try it, specify a list of sub-packages in the `workspaces` field of your `pac
 {
   "name": "my-project",
   "version": "1.0.0",
-  "workspaces": ["packages/*"]
+  "workspaces": ["packages/*"],
+  "devDependencies": {
+    "example-package-in-monorepo": "workspace:*"
+  }
 }
 ```
 
 {% callout %}
 **Glob support** — Bun supports simple `<directory>/*` globs in `"workspaces"`. Full glob syntax (e.g. `**` and `?`) is not yet supported.
 {% /callout %}
 
-This has a couple major benefits.
+When referencing other packages in the monorepo, use `"workspace:*"` as the version field in your `package.json`.
+
+```json
+{
+  "name": "pkg-a",
+  "version": "1.0.0",
+  "dependencies": {
+    "pkg-b": "workspace:*"
```

Comment:
can we avoid this `workspace:*` ?

Reply:
yeah, you can put `"pkg-b": "latest"` too